content/writing/digital-approaches-for-social-change-organisations.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "Digital approaches for social change organisations"
+date: 2021-09-13
+intro: Learnings from our collaboration with the Joseph Rowntree Foundation
+---
+
+## Context
+
+What does a social change organisation look like online? We recently worked with the [Joseph Rowntree Foundation](https://www.jrf.org.uk/) to explore this question through research and rapid prototyping.
+
+JRF's mission is to *inspire action and change to solve UK poverty*. They do research, make policy recommendations, co-design policies with people with lived experience of poverty, support grassroots organisations, and work in coalition with other charities and foundations. However, only a fraction of this work is currently visible online. While their research is fairly accessible to policy people and journalists, it could be made much more useful to grassroots campaigners.
```

Comment:
I know I always say this, but would have a new line between each logical idea as it makes it easier to read on the web. For example, I'd add one here at "However".